tools/mlir_to_bef.bzl

```diff
@@ -26,8 +26,8 @@ def mlir_to_bef(name, tfrt_translate):
     Returns:
       the name of generated bef file.
     """
-    bef_file = name[:-5] + ".bef"
-    rule_name = name + ".bef"
+    bef_file = name + ".bef"
+    rule_name = "bef_" + name
```

Comment:
Nit: `name + "_bef"` seems slightly better.

Reply:
For the rule_name? I have no strong preference, the reason I used "bef_" prefix is to "group" the BEF rules if sorted alphabetically, and to distinguish them clearly from the file name. But "_bef" suffix would work equally well (".bef" suffix doesn't work since bazel doesn't like it when the rule name and file names are the same)

Reply:
Yes, we often replace '.' with '_' to have the target (not rule, that's bad naming there) name not clash with the generated file. But let me try to land this as is.